sql/driver/tx.go

```diff
@@ -17,16 +17,39 @@
 
 package driver
 
-import "fmt"
+import "database/sql/driver"
 
 type tx struct {
 	conn *conn
 }
 
 func (t *tx) Commit() error {
-	return fmt.Errorf("TODO(pmattis): unimplemented")
+	_, err := t.Exec("COMMIT TRANSACTION")
```

Comment:
I believe this should be `t.conn.Exec("COMMIT TRANSACTION", nil)`. Now I'm remembering telling you something else in an earlier comment. Perhaps I led you astray.